### ISSUE TITLE
Poor testing data recon

### ISSUE THREAD

**Issue by aakash-saboo** [princeton-computational-imaging/Diffusion-SDF#26]
Hi.
I used the code to train on my cortical surface data. after 6 days of training, the training data recon is good, but it's not able to reconstruct the test data at all.
The loss is 0.0012, sdf loss: 0.00098 and vae loss: 1^-5.
Can you give any suggestions? Is it more about hyper-parameter tuning? My dataset have 600 surfaces.
@gene-chou

this is the test recon
![recon_bad](https://github.com/princeton-computational-imaging/Diffusion-SDF/assets/39874883/801c0c49-7e4b-4f85-a392-53a225400e5f)

and this is the training data recon.
![recon](https://github.com/princeton-computational-imaging/Diffusion-SDF/assets/39874883/22596281-8f6f-4ffc-8144-429bc6fb0e6a)

**Comment by gene-chou** [comment]
I think this data seems quite difficult to reconstruct. It's possible that the conditioning point cloud doesn't have enough information to reconstruct smooth surfaces. Additionally, see #25 regarding data.

**Comment by aakash-saboo** [comment]
Hey thanks for your response @gene-chou
I am using Pointcloud with 30K points. When visualizing it, they are quite clear with enough density. Since the model is able to reconstruct the training data well, I am assuming its a generalization issue.

by any chance did you face anything like this during your trainings?

**Comment by gene-chou** [comment]
Yes, generalization required either more data or more hyperparameter tuning. Later I had a chance to train on Objaverse and I was able to get good results without any tuning. With less data it can be quite difficult.

**Comment by aakash-saboo** [comment]
Hey 
Thanks!
I'll see if I can increase my dataset :)
